user_data_vault.sh.tpl

```diff
@@ -17,6 +17,52 @@ if [ "${audit_device}" = "true" ] ; then
   chmod 750 "${audit_device_path}"
 fi
 
+# Install, configure and initialize the AWS Cloudwatch agent
+if [ "${cloudwatch_agent}" = "true" ] ; then
+  yum install -y amazon-cloudwatch-agent
+
+  cat << EOF > /opt/aws/amazon-cloudwatch-agent/bin/config.json
+{
+        "agent": {
+                "metrics_collection_interval": 60,
+                "run_as_user": "root"
+        },
+        "metrics": {
+                "aggregation_dimensions": [
+                        [
+                                "InstanceId"
+                        ]
+                ],
+                "metrics_collected": {
+                        "disk": {
+                                "measurement": [
+                                        "used_percent"
+                                ],
+                                "metrics_collection_interval": 60,
+                                "resources": [
+                                        "/","/opt/vault"
```

Comment:
I'm not 100% sure; if this variable is mentioned in `variables.tf`, please use the `var.SOMETHING` here.